uk.co.compucorp.civicrm.tasksassignments/views/contact/documents.html

```diff
@@ -16,14 +16,17 @@
           <div class="col-xs-12 col-sm-4 col-lg-3">
             <div class="form-group" ng-if="settings.extEnabled.assignments">
               <label class="col-xs-12 col-sm-3 control-label">Type:</label>
-              <div class="col-xs-12 col-sm-9">
+              <div class="col-xs-11 col-sm-8">
                 <ui-select theme="civihr-ui-select" prevent-animations multiple ng-model="list.filterParams.assignmentType">
                   <ui-select-match prevent-animations class="ui-select-match" placeholder="Select workflow...">{{$item.title}}</ui-select-match>
                   <ui-select-choices prevent-animations class="ui-select-choices" repeat="type.id as type in cache.assignmentType.arr">
                     <div ng-bind-html="type.title | highlight: $select.search"></div>
                   </ui-select-choices>
                 </ui-select>
               </div>
+              <div class="col-xs-1 col-sm-1">
```

Comment:
I don't think you need "col-sm-1" class here, try to get rid of it and see if it works.